indra/llwebrtc/llwebrtc.cpp

```diff
@@ -246,10 +246,10 @@ void LLWebRTCImpl::init()
     apm_config.gain_controller1.enabled       = false;
     apm_config.gain_controller1.mode          = webrtc::AudioProcessing::Config::GainController1::kAdaptiveAnalog;
     apm_config.gain_controller2.enabled       = false;
-    apm_config.high_pass_filter.enabled       = true;
```

Comment:
This was reducing audio quality of streamed music.

Reply:
I suggest leaving simple coments in such cases, like:
apm_config.high_pass_filter.enabled = false; // off, because filter affects music